src/styles/card.css

```diff
@@ -0,0 +1,84 @@
+.card {
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  background: rgb(255, 255, 255);
+  max-width: 198px;
+  margin-bottom: 500px !important;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+}
+
+.card__image img {
+  width: 100%;
+}
+
+.card__title {
+  color: rgb(6, 11, 53);
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 17px;
+  padding-top: 40px;
+}
+
+.card__code {
+  color: rgb(97, 96, 112);
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  padding-top: 4px;
+}
+
+.card__review {
+  padding-top: 16px;
+  display: flex;
+  align-items: center;
+  margin-bottom: 24px;
+  justify-content: space-between;
+}
+
+.card__review-text {
+  font-size: 10px;
+  width: 53px;
+  text-align: right;
+}
+
+.card__price {
+  color: rgb(6, 11, 53);
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+  letter-spacing: 0%;
+  text-align: right;
+}
+
+.card__price-text {
+  color: rgb(97, 96, 112);
+  font-size: 12px;
+  line-height: 18px;
+  letter-spacing: 0%;
+}
+
+.card__value {
+  justify-content: space-between;
+  margin-bottom: 16px;
+  display: flex;
+}
+
+.card__button {
+  color: #fff;
+  background-color: #00acdc;
```

Comment:
The color `#00acdc` should be defined as an SCSS variable for better maintainability.